docs/cases/cases-ui-integrations.asciidoc

```diff
@@ -24,138 +24,88 @@ https://www.elastic.co/subscriptions[appropriate license], and your role needs *
-
-| *API token* | ({swimlane} only) The {swimlane} API authentication token is used for HTTP Basic authentication.
-This is the personal access token for your user role.
-
-|===
+** For {sn} connectors:
 +
-. Choose the connector type ({swimlane} only):
+IMPORTANT: If you've upgraded from {stack} version 7.15.0 or earlier to 7.16.0 or later, you must complete several prerequisites before creating a new {sn-itsm} or {sn-sir} connector. For more information, refer to prerequisites for {kibana-ref}/servicenow-action-type.html#servicenow-itsm-connector-prerequisites[{sn-itsm}] and {kibana-ref}/servicenow-sir-action-type.html#servicenow-sir-connector-prerequisites[{sn-sir}].
 +
-|===
 
-| *All* | You can choose to set all or no field mappings when creating your new {swimlane} connector. However, note that if
-you don’t set field mappings now, you’ll be prompted to do so if you want to use the connector for a case or a rule.
-
-| *Alerts* | Provide an alert ID and rule name.
-
-| *Cases* | Provide a case ID, a case name, comments, and a description.
-
-|===
+* *{sn} instance URL*: The URL of the {sn} instance to which you want to send cases.
+* *Use OAuth authentication*: Enable this to use open authorization (OAuth) to authenticate a connection between Elastic and {sn}.
 +
-. Save the connector.
+NOTE: To use open authorization (OAuth), you must {kibana-ref}/servicenow-action-type.html#servicenow-itsm-connector-prerequisites-rsa-key[create an RSA keypair and add an X.509 Certificate] and also {kibana-ref}/servicenow-action-type.html#servicenow-itsm-connector-prerequisites-endpoint[create an OAuth JWT API endpoint for external clients with a JWT Verifiers Map.]
++
+
+* *Username*: (Displays if *Use OAuth authentication* is turned _off_.) The username of the {sn} account used to access the {sn} instance.
+* *Password*: (Displays if *Use OAuth authentication* is turned _off_.) The password of the {sn} account used to access the {sn} instance.
+* *Client ID*: (Displays if *Use OAuth authentication* is turned _on_.) The client ID assigned to your OAuth application.
+* *User Identifier*: (Displays if *Use OAuth authentication* is turned _on_.) Identifier to use for OAuth type authentication. Use the value you entered into the *User field* when you created an OAuth JWT API endpoint for external clients.
+* *JWT Verifier Key ID*: (Displays if *Use OAuth authentication* is turned _on_.)
+The key ID assigned to the JWT Verifier Map of your OAuth application.
+* *Client Secret*: (Displays if *Use OAuth authentication* is turned _on_.) The client secret assigned to your OAuth application.
+* *Private Key*: (Displays if *Use OAuth authentication* is turned _on_.)
+The RSA private key generated when you created an RSA keypair.
+* *Private Key Password*: (Displays only if *Use OAuth authentication* is turned _on_.) The password for the RSA private key generated during setup, if set.
```

Comment:
To match the other items in the list, and I think "only" is unnecessary since "if" on its own establishes the conditionality.
```suggestion
* *Private Key Password*: (Displays if *Use OAuth authentication* is turned _on_.) The password for the RSA private key generated during setup, if set.
```

Reply:
Good catch - thanks!